metron-platform/metron-writer/src/main/java/org/apache/metron/writer/hdfs/HdfsWriter.java

```diff
@@ -154,13 +170,16 @@ synchronized SourceHandler getSourceHandler(String sourceType, String stellarRes
     SourceHandler ret = sourceHandlerMap.get(key);
     if(ret == null) {
       if(sourceHandlerMap.size() >= maxOpenFiles) {
-        throw new IllegalStateException("Too many HDFS files open!");
+        String errorMsg = "Too many HDFS files open! Maximum number of open files is: " + maxOpenFiles;
```

Comment:
Another useful addition to our logging.  Can you also add the number of files that are open too?  

I know it would normally be close to, if not equal to the max, but weirder things have happened and that's what good logging is there to help with.